app/index.js

```diff
@@ -1,10 +1,10 @@
-var React = require('react');
-var ReactDOM = require('react-dom');
-require('./style.css');
-const App = require('./components/App');
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Import only what you need: `import { render, } from 'react-dom'`

Reply:
BTW, `import React from 'react' is a very rare exception to that rule. You can read about why, but it's more important to know that it is.

Reply:
done